flake.nix

```diff
@@ -0,0 +1,43 @@
+{
+  description = "Faucet for Cardano";
+  inputs = {
+    nixpkgs.url = "github:NixOS/nixpkgs/d0f9857448e77df50d1e0b518ba0e835b797532a";
```

Comment:
This seems highly risky. We should _not_ update to random nixpkgs pins, this is likely going to break frequently.

Reply:
This is leftover from me troubleshooting 😄 I was just pushing the PR to get some help, it's not in a pretty state.